cmd/argo-watcher/state/postgres_state_test.go

```diff
@@ -110,12 +119,17 @@ func TestPostgresState_GetTask(t *testing.T) {
 		t.Errorf("got error %s, expected nil", err.Error())
 	}
 
-	assert.Equal(t, "in progress", task.Status)
+	assert.Equal(t, task.Status, models.StatusInProgressMessage)
```

Comment:
I believe that the order is wrong. It should be `assert.Equal(t, expected, actual)`.